docs/configuration/initialization.md

```diff
@@ -0,0 +1,129 @@
+---
+id: initialization
+title: Initialization
+---
+
+In Next.js, you can define an API route that will catch all requests that begin with a certain path. Conveniently, this is called [Catch all API routes](https://nextjs.org/docs/api-routes/dynamic-api-routes#catch-all-api-routes).
+
+When you define a `/pages/api/[...nextauth]` JS/TS file, you instruct NextAuth.js that every API request beginning with `/api/auth/*` should be handled by the code written in the `[...nextauth]` file.
+
+Depending on your use-case, you can initialize NextAuth.js in two different ways:
+
+## Simple initialization
+
+In most cases, you won't need to worry about what `NextAuth.js` does, and you will get by just fine with the following initialization:
+
+```ts title="/pages/api/auth/[...nextauth].js"
+import NextAuth from "next-auth"
+
+export default NextAuth({
+  ...
+})
+```
+
+Here, you only need to pass your [options](/configuration/options) to `NextAuth`, and `NextAuth` does the rest.
+
+This is the preferred initialization in tutorials/other parts of the documentation, as it somewhat simplifies the code and there is smaller chances for errors in the authentication flow.
```

Comment:
How about to simplify this sentence, "..as it simplifies the code and reduces potential errors in the authentication flow" ?